### ISSUE TITLE
:obj:`.'` in the documentation rendered as ', without the dot.

### ISSUE THREAD

**Issue by tpapp** [JuliaLang/julia#16259]
In the [noteworthy differences](https://github.com/JuliaLang/julia/blob/master/doc/manual/noteworthy-differences.rst) section, in the line
```rst
- Julia performs matrix transposition using the :obj:`.'` operator and conjugated
  transposition using the :obj:`'` operator. Julia's ``A.'`` is therefore
  equivalent to R's ``t(A)``.
```
the `.'` is rendered without the dot when the documentation is generated (eg [HTML](http://docs.julialang.org/en/release-0.4/manual/noteworthy-differences/#noteworthy-differences-from-r)).

**Comment by tkelman** [comment]
may need extra escaping

**Comment by jgrg** [comment]
I worked out how to do the escaping and submitted a pull request: #16665

**Comment by tkelman** [comment]
thanks for the bump @jgrg, closed by #16665 - let us know after readthedocs renders a copy with these changes
